index.html

```diff
@@ -73,7 +75,7 @@ <h1>Our Best Features</h1>
                 </div>
                 <div class="conteudo2-boards">
                     <div class="boards">
-                        <i></i>
+                        <img src="icone.svg" alt="">
```

Comment:
Voce poderia adicionar o icone para que o codigo funcione corretamente? 
```suggestion
                        <img src="icone.svg" alt="">
```